### PR TITLE
Umbraco Cloud Release notes for February

### PR DESCRIPTION
**Pull request by sajumb**
## Description

Add new page for February release notes for Cloud

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ X ] New content
* [ ] Updates related to a new version
* [ ] Other

## Deadline (if relevant)

As soon as possible

### CODE REVIEW

**Comment by alina-tincas** [comment]
Hi @sajumb thank you for the PR 🙌 

We will review it as soon as we can 😊

File: umbraco-cloud/release-notes/2024-02-releasenotes.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+Each entry in the overview is interactive, allowing you to click on it for more detailed information. This feature ensures that you have easy access to in-depth details about each event, further enhancing your insight into your project's history and ongoing activities.
+
+The overview today concentrates on activities spurred by users and Umbraco Cloud. However, our vision for the future is more expansive. As we diligently work on enhancing and refining this feature, the "Project History Page" is set to evolve into an all-encompassing overview. It will offer an unparalleled glimpse into the entirety of events that have transpired throughout your cloud project's existence.
+
+## New Upgrade Details Page
+
+The new "Upgrade Details" page is your gateway to detailed insights for every upgrade. By clicking the 'info' link from the "Project History" page for a manual or automatic upgrade, you'll land on a page that not only highlights the essentials—product version, precise timestamps, current statuses, and error reports for troubleshooting—but also showcases the impact of these upgrades across all relevant environments for your project.
```
Comment (jonat123) [inline umbraco-cloud/release-notes/2024-02-releasenotes.md:20]:
```suggestion
The new "Upgrade Details" page is your gateway to detailed insights for every upgrade. By clicking the 'info' link from the "Project History" page for manual or automatic upgrades, you'll land on a page that highlights the essentials—product version, precise timestamps, current statuses, and error reports for troubleshooting. The page also showcases the impact of these upgrades across all relevant environments for your project.
```

File: umbraco-cloud/release-notes/2024-02-releasenotes.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+We're here to support your journey every step of the way. Should you have any questions or need further assistance, feel free to reach out. Happy upgrading!
+
+## Improved Search on Projects Dashboard
+Finally, we have rolled out a significant update to the search functionality on the Projects Dashboard, making it easier and faster for you to navigate your projects. Now, you're not limited to searching by project names. This enhancement introduces the ability to search using a wide array of associated tags and even specific product versions within each project's environment. Examples of searchable tags include project type, project plan, whether the project is a baseline or a child, and payment type, providing a multifaceted approach to organizing and locating your projects.
```
Comment (jonat123) [inline umbraco-cloud/release-notes/2024-02-releasenotes.md:29]:
```suggestion
Finally, we have rolled out a significant update to the search functionality on the Projects Dashboard. The update makes it easier and faster for you to navigate your projects. Now, you're not limited to only searching by project names. This enhancement introduces the ability to search using a wide array of associated tags and even specific product versions within each project's environment. 

Examples of searchable tags include:

* Project type.
* Project plans.
* Payment type. 
* If the project is a baseline or a child.

Providing a multifaceted approach to organizing and locating your projects.
```

**Review by jonat123 — APPROVED** [review]
@sajumb Looks good.

merging in.

**Pull request merged**